pkgs/by-name/se/seventeenlands/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  fetchPypi,
+  lib,
+  python3,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "seventeenlands";
+  version = "0.1.42";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-P/imV4vvyd6wgjqXzgfIAURFtFhLwX1eS8eiPl79oZk=";
+  };
+
+  # No tests
+  doCheck = false;
```

Comment:
Then at least add a `pythonImportsCheck = [ "seventeenlands" ];` (test if it works)